dhis-2/dhis-test-web-api/src/test/java/org/hisp/dhis/webapi/controller/programstageworkinglist/ProgramStageWorkingListControllerTest.java

```diff
@@ -261,8 +267,26 @@ void shouldDeleteWorkingListWhenDeleteRequested() {
     assertEquals(HttpStatus.OK, response.status());
   }
 
-  @Test
-  void shouldCreateWorkingListWithProgramStageQueryCriteria() {
+  static Stream<Arguments> shouldCreateWorkingListWithProgramStageQueryCriteria() {
+    Consumer<JsonProgramStageQueryCriteria> followUpIsNull =
+        json ->
+            assertFalse(
+                json.has("followUp"),
+                "FollowUp check has not been requested and should not be in the response");
+    Consumer<JsonProgramStageQueryCriteria> followUpIsTrue = json -> assertTrue(json.getFollowUp());
+    Consumer<JsonProgramStageQueryCriteria> followUpIsFalse =
+        json -> assertFalse(json.getFollowUp());
+
+    return Stream.of(
+        arguments("", followUpIsNull),
+        arguments("'followUp': 'true',\n", followUpIsTrue),
```

Comment:
should this be a string or `true`/`false` literal?

Reply:
ok